### ISSUE TITLE
No fall back for share dialog

### ISSUE THREAD

**Issue by theerapat-y** [mokesmokes/titanium-ios-facebook#23]
I see that the share dialog now requires the Facebook app [from 3.20.02](https://github.com/mokesmokes/titanium-ios-facebook/commit/a4333a25b0e67ab9f090509f36c5320375895b25#diff-835c1cb2306e65390d461aed03eceb32R530).
Although on the Facebook SDK, fall back sharing using feed dialog is possible. https://developers.facebook.com/docs/ios/share#linkfeed

**Comment by mokesmokes** [comment]
That's correct... however the feed dialog requires a webview login which is a pain. Do you really think a user who didn't install the Facebook app is the type of user interested in sharing her experience on FB? What I do in my app is hide the share button if the app is not installed (getCanPresentShareDialog is false), I think it's a much better user experience than a webview fallback. 

In my eyes all the webview methods are very low priority for this project, and indeed I can't think of a popular app today which uses them.
